docs/src/components/home/sectionHeader.tsx

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+
+import "@site/src/css/home/sectionHeader.css";
+
+/**
+ * {@link SectionHeader} component props.
+ */
+export interface SectionHeaderProps {
+	/**
+	 * Section title.
+	 */
+	title: string;
+
+	/**
+	 * Optional section subtitle.
+	 */
+	subtitle?: string;
+}
+
+/**
+ * Homepage section header component.
+ */
+export function SectionHeader({ title, subtitle }: SectionHeaderProps): JSX.Element {
+	return (
+		<div className="ffcom-section-header">
+			<div className="ffcom-section-header-inner">
+				{subtitle === undefined ? (
```

Comment:
Is it correct that subtitle is rendered above the title?

Reply:
Yeah, that was the design spec we were given.